src/compiler/utilities.ts

```diff
@@ -3563,6 +3566,15 @@ export function getJSDocCommentsAndTags(hostNode: Node, noCache?: boolean): read
             result = addRange(result, (noCache ? getJSDocTypeParameterTagsNoCache : getJSDocTypeParameterTags)(node as TypeParameterDeclaration));
             break;
         }
+        if (isBindingElement(node) && isIdentifier(node.name)) {
+            const root = getRootDeclaration(node);
+            if (isParameter(root) && isObjectBindingPattern(root.name)) {

```

Comment:
Hm, it would be nice to support documenting binding elements on variable declarations too. But I don't know which tag to use.